src/index.html

```diff
@@ -1,22 +1,474 @@
+<!-- #region head -->
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, initial-scale=1.0"
+      content="widt64px !important;ice-width, initial-scale=1.0"
     />
-    <title>Landing Page</title>
+    <title>Bang & Olufsen</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Manrope:wght@200..800&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
-      href="./styles/main.scss"
+      href="./styles/style.scss"
     />
+
     <script
       src="scripts/main.js"
       defer
     ></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+
+            <a
+              href="#"
+              class="top-bar__logo-link"
+            >
+              <img
+                src="images/bang&olufsen_logo.svg"
+                alt="Bang & Olufsen"
+                class="top-bar__logo"
+              />
+            </a>
+
+            <div class="top-bar__icons">
+              <a
+                href="#"
+                class="icon icon--phone"
+              ></a>
+              <a
+                href="#menu"
+                class="icon icon--menu"
+              ></a>
+            </div>
+
+          </div>
+
+          <div class="header__bottom">
+            <h1 class="header__title">
+              Your senses.
+              <br />
+              Set free.
+            </h1>
+          </div>
+
+        </div>
+      </div>
+    </header>
+
+    <aside
+      class="page__menu menu"
+      id="menu"
+    >
+      <div class="container">
+        <div class="top-bar menu__top">
+
+          <a href="#">
+            <img
+              class="top-bar__logo"
+              src="images/bang&olufsen_logo.svg"
+              alt="Bang & Olufsen"
+            />
+          </a>
+
+          <a
+            href="#"
+            class="icon icon--close"
+          ></a>
+
+        </div>
+
+        <div class="menu__bottom">
+          <nav class="nav menu__nav">
+
+            <ul class="nav__list">
+
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="#recommended"
```

Comment:
ur links are broken) I think u forgot to add `id` in each section.
after clicking the link in the nav.panel, we should be moved to the selected section
Check [Loom video](https://www.loom.com/share/6a0c14db9acb47cd89689fd9da17eb8b)